src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/SpringAPIServiceTest.kt

```diff
@@ -47,25 +48,26 @@ import kotlin.annotation.AnnotationTarget.CLASS
 @MustBeDocumented
 @Inherited
 @ExtendWith(
-  NomisApiExtension::class,
-  MappingExtension::class,
   HmppsAuthApiExtension::class,
-  VisitsApiExtension::class,
-  IncentivesApiExtension::class,
```

Comment:
ordered alphabetically (with exception of common HmppsAuthApiExtension/MappingExtension/NomisApiExtension)